test/browser/spec/ol/renderer/canvas/Layer.test.js

```diff
@@ -16,7 +16,7 @@ describe('ol/renderer/canvas/Layer', function () {
       layers: [
         new TileLayer({
           source: new XYZ({
-            url: 'spec/ol/data/osm/{z}-{x}-{y}.png',
+            url: 'spec/ol/data/osm-{z}-{x}-{y}.png',
```

Comment:
This is not strictly related, but I found the typo while investigating the tile layer renderer problem.